modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc

```diff
@@ -188,37 +193,48 @@ void LaneChangeDecider::UpdateStatus(double timestamp,
 }
 
 void LaneChangeDecider::PrioritizeChangeLane(
-    std::list<ReferenceLineInfo>* reference_line_info) const {
+    std::list<ReferenceLineInfo>* reference_line_info,
+    const bool is_prioritize_change_lane) const {
```

Comment:
Put input param before output param. If the input list is empty, prefer
```
std::list<ReferenceLineInfo> LaneChangeDecider::PrioritizeChangeLane(const bool is_prioritize_change_lane) const;
```
which is even cleaner.

Reply:
fixed. thank you!